src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/MessagePack/Formatters/RazorConfigurationFormatter.cs

```diff
@@ -40,14 +41,19 @@ public override RazorConfiguration Deserialize(ref MessagePackReader reader, Ser
             ? version
             : RazorLanguageVersion.Version_2_1;
 
-        return new(languageVersion, configurationName, extensions, SuppressAddComponentParameter: suppressAddComponentParameter);
+        return new(
+            languageVersion,
+            configurationName,
+            extensions,
+            UseConsolidatedMvcViews: useConsolidatedMvcViews,
+            SuppressAddComponentParameter: suppressAddComponentParameter);
```

Comment:
nit: It looks like these are in the correct order, so the named arguments can be elided.

Reply:
Ah good catch. I was changing the ordering to help exactly like this